Remotion/Data/DomainObjects.UnitTests/IntegrationTests/Synchronization/VirtualEndPointGarbageCollectionTest.cs

```diff
@@ -72,7 +72,12 @@ public void UnloadUnsynchronizedFK_LeavesCompleteEmptyCollection ()
       var unsynchronizedSubordinateID =
           RelationInconcsistenciesTestHelper.CreateObjectAndSetRelationInOtherTransaction<Employee, Employee>(
             employee.ID,
-            (subOrdinate, e) => subOrdinate.Supervisor = e);
```

Comment:
marked file (please move property initialization up to object load/initialization)

Reply:
I can move the initialization of the existing object out of this delegate, but since `CreateObjectAndSetRelationInOtherTransaction` creates an object generically, I have to initialize that object in a delegate given to the generic method.
Can I use this delegate, should I create a separate one, or do we rework the whole approach to DomainObject creation in the tests?